### ISSUE TITLE
Cant working

### ISSUE THREAD

**Issue by miamor028** [imlakshay08/Waste-Management-System#5]
I try to put all codes in VSC then i try to import wms.sql but still notworking😔 Please help

**Comment by miamor028** [comment]
Only index.html working but the complain, login, sign,up, preview complain not working

**Comment by imlakshay08** [comment]
Thank you for raising this issue.

Did you followed this guide for [setting up the project](https://github.com/imlakshay08/waste-management-system#how-to-setup-this-project-locally) correctly ? 

> I try to put all codes in VSC then i try to import wms.sql but still notworking😔 Please help

Also, just putting all code in VS code won't work. You need to install XAMPP/WAMP server on your system. You could download that from [here](https://www.apachefriends.org/download.html).

**Comment by miamor028** [comment]
yes I followed, but still didn't working also other functions not working/

**Comment by miamor028** [comment]
![1700984012231582764093082151073](https://github.com/imlakshay08/waste-management-system/assets/151865609/6cdaa1ec-01a9-42a0-8ce0-b6d9e216e61e)

When i click open this is what comes up, even though xampp is open and i import it.

**Comment by imlakshay08** [comment]
@miamor028 , since this is a dynamic project and not static, you need to host it locally on a server. Make sure, Apache and MySQL (which are provided by xampp) server are up and running. To make sure your server is running, you could navigate to [http://localhost/](http://localhost/) or [http://localhost:8080/](http://localhost:8080/) in your browser (here, 8080 is port on which the server is running, you could change this depending on which port your server is running in xampp control panel itself). After opening this localhost, you might see the XAMPP homepage, something like [this](https://www.apachefriends.org/images/screenshots/xampp-windows-start-b7cf46ea.jpg). 

First, Let me know you have achieved this much.

**Comment by miamor028** [comment]
I followed everything, I opened xampp (apache and mysql) and I imported the wms.sql, then I tried to open it locally which is localhost/waste-management-system. The landing page is working but when I click on complain nothing happens, I can't complain and I can't preview complain even though I can't open the admin one, the dashboards can't really open. Because the codes are missing 🙃

**Comment by imlakshay08** [comment]
Where you are importing the wms.sql file exactly? And how are you importing it?

**Comment by kasar123** [comment]
Warning: mysqli_connect(): (HY000/1045): Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\WMS\connection.php on line 2

Fatal error: Uncaught TypeError: mysqli_real_escape_string(): Argument #1 ($mysql) must be of type mysqli, bool given in C:\xampp\htdocs\WMS\controllerUserData.php:10 Stack trace: #0 C:\xampp\htdocs\WMS\controllerUserData.php(10): mysqli_real_escape_string(false, 'ssik') #1 C:\xampp\htdocs\WMS\signup-user.php(1): require_once('C:\\xampp\\htdocs...') #2 {main} thrown in C:\xampp\htdocs\WMS\controllerUserData.php on line 10

**Comment by kasar123** [comment]
this is a prob
